packages/react-native/sdks/hermes-engine/utils/build-apple-framework.sh

```diff
@@ -52,6 +52,10 @@ function get_visionos_deployment_target {
   use_env_var "${XROS_DEPLOYMENT_TARGET}" "XROS_DEPLOYMENT_TARGET"
 }
 
+function get_tvos_deployment_target {
+  use_env_var_or_ruby_prop "${IOS_DEPLOYMENT_TARGET}" "deployment_target('tvos')"
+}
+
```

Comment:
can we use the same approach we used below? We found that reading the podspec is unreliable in GitHub actions, so now we prefer to be explicit with the ENV variables.

Reply:
Since we use the same deployment target for both iOS and tvOS, I removed this function as no longer needed.